### PR TITLE
Backport dealer-navigation style changes

### PR DESCRIPTION
**Pull request by omakoleg**
/cc @AutoScout24/web-experience

## Description

Fix some issues with `sc-navigation` and add additional features: `subtitle` and `li.separator` which present in internal navigation

## Risks
- medium - CXP navigation could break

## Problems

Local tests failed with:
```
31 07 2019 14:43:05.514:ERROR [launcher]: Cannot load browser "bs_chrome_win"!
  Error: Username is required.
31 07 2019 14:43:05.515:ERROR [launcher]: Cannot load browser "bs_firefox_win"!
```
Also was attempt to publish new version to NPM (possibly git hook triggered) when I ve pushing to my remote.
```
npm ERR! 401 Unauthorized - PUT https://registry.npmjs.org/showcar-ui - You must be logged in to publish packages.
```

### CODE REVIEW

File: src/06-components/organisms/navigation/navigation.scss [hunk 1]
```diff
@@ -63,16 +63,19 @@
   }
 
   nav {
+    z-index: 130;
     transition: max-height 0.5s ease-out;
     max-height: 0;
     overflow: hidden;
 
     & > ul {
+      margin-top: 13px;
```
Comment (rowantower) [inline src/06-components/organisms/navigation/navigation.scss]: can we use $M (12px) here?

File: src/06-components/organisms/navigation/navigation.scss [hunk 2]
```diff
@@ -214,24 +226,90 @@
...
+
+        &:last-child {
+          padding-bottom: $L;
+        }
+
+        /* direct children on top level */
+        > li {
+          /* draw line between top level li */
+          &:before {
+            margin: 8px 0;
```
Comment (rowantower) [inline src/06-components/organisms/navigation/navigation.scss]: 8px => $S

File: src/06-components/organisms/navigation/navigation.scss [hunk 3]
```diff
@@ -214,24 +226,90 @@
...
+            display: block;
+            border-top: 1px solid #dcdcdc;
+            content: ' ';
+          }
+        }
+      }
+
+      /* Adjust icon opening submenu */
+      span.title:after {
+        margin-right: 16px;
```
Comment (rowantower) [inline src/06-components/organisms/navigation/navigation.scss]: 16px => $L

File: src/06-components/organisms/navigation/navigation.scss [hunk 4]
```diff
@@ -327,6 +406,28 @@
...
+            right: 0;
+            left: auto;
+          }
+        }
+
+        /* Special empty li separator */
+        li.separator {
+          box-sizing: border-box;
+          height: 1px;
+          margin: 12px 0;
```
Comment (rowantower) [inline src/06-components/organisms/navigation/navigation.scss]: 12px => $M

**Comment by rowantower** [comment]
You were probably running not the local tests with karma (`gulp test` or `gulp test:fast`) but with `npm test` that runs test on travis. That's why you have problems with credentials for browser stack.
I ran `gulp test` on your pr and it's all green ✔️

**Comment by rowantower** [comment]
Yes, the output you saw is the githook and it only work if you have npm credentials. So don't worry about it.

File: src/06-components/organisms/navigation/navigation.scss (hunk 1 above)
Comment (omakoleg) [reply src/06-components/organisms/navigation/navigation.scss]: yes

**Comment by omakoleg** [comment]
Responsive point also updated from `9xx`px to `$viewportL` d'ont know would it help in my specific use case (would all fit in or not).

**Pull request merged**